### ISSUE TITLE
[Feature request] [G19] Integrate clock in the background

### ISSUE THREAD

**Issue by Eagle3386** [whatpulse/WhatPulse-Logitech-Widget#1]
Most built-in widgets from Logitech aren't that cool for me, but I do love that coloured clock which runs on my LCD all the time - up until now.

Thanks to you guys, I'm finally able to watch my stats in a cool manner. But having to choose between the clock and your widget is kinda having feet of clay as the saying goes.

Therefore I'd like to request that a clock gets put into the simple black background. I'll try myself in a fork, but since I'm no experienced C(++) dev, I'll be having a hard time doing that. So if someone who's more qualified drops by, maybe he/she can implement that feature better and faster than me.

**Comment by smitmartijn** [comment]
Pretty good idea..I'll put this on my todo list. Considering that's a long list, if anyone wants to do it before I get to it, you're welcome to do so. :-)

**Comment by Eagle3386** [comment]
It's been almost 2 years, hence I'd like to ask about progress on your todo list, so that this idea gets implemented? :)

**Comment by smitmartijn** [comment]
Yeah, so about the WhatPulse todo list...It’s miles long. 😉 Plus there are priorities, which are mostly on the clients and website. Long answer for “I haven’t gotten around to it” - if you are still willing to try, go for it. I’ll be of help in any way I can.

**Comment by Eagle3386** [comment]
Sorry, but I couldn't reply sooner..

Unfortunately, I'm unable to go for it as I'm still no C++ developer beyond "simple fixes" and that won't change anytime soon (doing mostly C#/Web stuff). :(
Also, I believe that it's quite hard to do, because it'd require kind of an "injection" of WhatPulse's stats into the G19's clock screen.

**Comment by smitmartijn** [comment]
It shouldn't have to be that hard; assuming a textual clock is fine. You'd either add another text field with the current time or replace on of the current fields with the current time.
